daemon/graphdriver/lcow/lcow.go

```diff
@@ -537,3 +900,25 @@ func (d *Driver) setLayerChain(id string, chain []string) error {
 	}
 	return nil
 }
+
+// getLayerDetails is a utility for getting a file name, size and indication of
+// sandbox for a VHD(x) in a folder. A read-only layer will be layer.vhd. A
+// read-write layer will be sandbox.vhdx.
+func getLayerDetails(folder string) (string, int64, bool, error) {
```

Comment:
Yeah, let me do that in the current branch I'm working in for a future PR which builds on the lifetime changes in this PR. Will avoid a confusing rebase.